global.go

```diff
@@ -67,6 +67,17 @@ func ReplaceGlobals(logger *Logger) func() {
 	return func() { ReplaceGlobals(prev) }
 }
 
+// NewStdLog returns a standard logger which writes to the supplied zap logger at InfoLevel.
+func NewStdLog(l *Logger) *log.Logger {
+	const (
+		stdLogDefaultDepth = 4
+		loggerWriterDepth  = 1
+	)
```

Comment:
Now that we're using these constants in two places, let's make them global. Please remember that this project's style underscore-prefixes globals, so this will be 

```
const (
  _stdLogDefaultDepth = 4
  _loggerWriterDepth = 1
)
```